pkg/constants/constants.go

```diff
@@ -176,7 +176,7 @@ const (
 	RootfsAsset = "rootfs.sqsh"
 
 	// NodeCertFile is the filename where the current Talos Node Certificate may be found
-	NodeCertFile = "/var/talos-node.crt"
+	NodeCertFile = "/run/talos-node.crt"
```

Comment:
This is one less file in `/var`, and we wanted to make node certs ephemeral so figured I could add it here. I will submit another PR so that we have it in the changelog, feels important enough.